index.html

```diff
@@ -452,6 +467,12 @@ <h3 class="card__title">Comments</h3>
     </footer>
     <script>
       const myForm = document.getElementById("form");
+      const cards = form.querySelectorAll("input#experienceUrl");
+      const buttonsToDisable = [
+        myForm.querySelector('#siteUrl'),
+        myForm.querySelector('#technologyWeb'),
+        myForm.querySelector('#technologyAmp')
+      ];
```

Comment:
I know there are other variables defined here but, since those are used only by `toggleFormStatus`, I'd define them there.

```suggestion
```

Reply:
I put them outside so that we wouldn't be executing the querySelector so many times as the toggleFormStatus is executed every time that we change selection but I guess that that's not the usual behaviour of the user

Reply:
Since we don't have many elements, I think it wouldn't be an issue but It's ok to keep it here anyway. 👍